docs/main/prerequesites.md

```diff
@@ -53,6 +53,7 @@ Due to the complexity of different ways of setting an IP address across differen
 ### Firewalls
 
 Below are some examples of firewall rules that will need to be set on your Pi-hole server in order to use the functions available. These are only shown as guides, the actual commands used will be found with your distribution's documentation.
+Because Pi-hole was born to work inside a local network, the following rules will block the traffic from the Internet for security reasons. `192.168.0.0/16` is the most common local network IP range for home users but it can be different in your case. Check your local network settings before applying these rules. Other common local network IPs are `10.0.0.0/8` and `172.16.0.0/12`.
```

Comment:
`Check your local network settings before applying these rules.` Can this be very prominent please? Bold, broken out to a single sentence by itself, <marquee/>... Anything to make it be the takeaway from the paragraph.